pytential/linalg/utils.py

```diff
@@ -402,6 +426,23 @@ def skeletonization_error(
         mat: np.ndarray, skeleton: "SkeletonizationResult", *,
         ord: Optional[float] = None,
         relative: bool = False) -> np.ndarray:
+    r"""Computes the skeletonization error for the entire matrix *mat*.
+
+    The error computed here is given by
+
+    .. math::
+
+        E = \|A - L S R\|,
```

Comment:
Specify the norm being used, as above.

Reply:
https://github.com/inducer/pytential/compare/440161e74f559e51120245ce59f8c1a6c4f15d77..0617bdafd0a7a37c2e162b752371ab908e456ad5